src/HostManager.cpp

```diff
@@ -166,3 +166,21 @@ Host * HostManager::getFirstHost()
     QMutexLocker locker(& mPoolLock);
     return mHostPool.begin().value().data();
 }
+
+Host * HostManager::getNextHost( QString lastHost )
```

Comment:
What is the point of this function? The hosts are in a QMap, which is alphabetical so any specific ordering is lost.